x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/overview_logic.test.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resetContext } from 'kea';
+import { act } from 'react-dom/test-utils';
+
+import { mockKibanaContext } from '../../../__mocks__';
+
+import { mockLogicValues } from './__mocks__';
+import { OverviewLogic } from './overview_logic';
+
+describe('OverviewLogic', () => {
+  let unmount: any;
+
+  beforeEach(() => {
+    resetContext({});
+    unmount = OverviewLogic.mount() as any;
+    jest.clearAllMocks();
+  });
+
+  afterEach(() => {
+    unmount();
+  });
+
+  it('has expected default values', () => {
+    expect(OverviewLogic.values).toEqual(mockLogicValues);
+  });
+
+  describe('setServerData', () => {
+    const feed = [{ foo: 'bar' }] as any;
+    const user = { firstName: 'Joe', email: 'e@e.e', name: 'Joe Jo', color: 'pearl' };
+    const account = {
+      name: 'Jane doe',
+      id: '1243',
+      isAdmin: true,
+      canCreatePersonalSources: true,
+      groups: [],
+      supportEligible: true,
+    };
+    const org = { name: 'ACME', defaultOrgName: 'Org' };
+
+    const data = {
+      accountsCount: 1,
+      activityFeed: feed,
+      canCreateContentSources: true,
+      canCreateInvitations: true,
+      currentUser: user,
+      fpAccount: account,
+      hasOrgSources: true,
+      hasUsers: true,
+      isFederatedAuth: false,
+      isOldAccount: true,
+      organization: org,
+      pendingInvitationsCount: 1,
+      personalSourcesCount: 1,
+      sourcesCount: 1,
+    };
+
+    beforeEach(() => {
+      OverviewLogic.actions.setServerData(data);
+    });
+
+    it('will set `dataLoading` to false', () => {
+      expect(OverviewLogic.values.dataLoading).toEqual(false);
+    });
+
+    it('will set server values', () => {
+      expect(OverviewLogic.values.organization).toEqual(org);
+      expect(OverviewLogic.values.isFederatedAuth).toEqual(false);
+      expect(OverviewLogic.values.currentUser).toEqual(user);
+      expect(OverviewLogic.values.fpAccount).toEqual(account);
+      expect(OverviewLogic.values.canCreateInvitations).toEqual(true);
+      expect(OverviewLogic.values.hasUsers).toEqual(true);
+      expect(OverviewLogic.values.hasOrgSources).toEqual(true);
+      expect(OverviewLogic.values.canCreateContentSources).toEqual(true);
+      expect(OverviewLogic.values.isOldAccount).toEqual(true);
+      expect(OverviewLogic.values.sourcesCount).toEqual(1);
```

Comment:
We're missing some values:

```suggestion
      expect(OverviewLogic.values.sourcesCount).toEqual(1);
      expect(OverviewLogic.values.pendingInvitationsCount).toEqual(1);
      expect(OverviewLogic.values.accountsCount).toEqual(1);
      expect(OverviewLogic.values.personalSourcesCount).toEqual(1);
      expect(OverviewLogic.values.activityFeed).toEqual(feed);
```